add definition for colonialism (#343)
selfdefined/web-app: 11ty/definitions/colonialism.md

---

> title: Colonialism

Reviewer: ```suggestion
title: colonialism
```

slug: colonialism
speech: noun

> excerpt: The act of enforcing one's political and ideological dominance in a country or region that is not of one's origin

Reviewer: ```suggestion
excerpt: the act of enforcing one's political and ideological dominance in a country or region that is not of one's origin
```

sub_terms:

>     - text: neo colonialism, western colonialism, modern colonialism

Reviewer: ```suggestion
    - text: modern
    full_title: modern colonialism
    - text: neo-
    full_title: neo-colonialism
    - text: settler
    full_title: settler colonialism
    - text: Western
    full_title: Western colonialism
```

defined: true
reading:
    - text: A brief context of colonialism
      href: https://plato.stanford.edu/entries/colonialism/
---

> Colonialism is the act of forcing a group of people in a region or country to live under the policies and ideologies of an invading government or group of people for the purposes of asserting geo-political dominance. Though [often associated with imperialism](https://plato.stanford.edu/entries/colonialism/), the two words have different latin etymologies that drive how the connotation of colonialism and the word colony have evolved. The most notable instances of colonialism to the general public are western colonialism and recently, neo-colonialism.

Reviewer: ```suggestion
the act of forcing a group of people in a region or country to live under the policies and ideologies of an invading government or group of people for the purposes of asserting geo-political dominance.

### Note
Though [often associated with imperialism](https://plato.stanford.edu/entries/colonialism/), the two words have different Latin etymologies that drive how the connotation of colonialism and the word colony have evolved. The most notable instances of colonialism to the general public are Western colonialism and recently, neo-colonialism.

```

> The practice of colonialism was most prominent with the British, Ottoman, and other European empires upon regions in the Americas, Asia, and Africa where slave trade was implemented as an economic vehicle. Contrary to popular belief, the colonialism also affected people of European descent, most notably, the Irish. Some studies have tried to [clarify the question](https://www.jstor.org/stable/20720270?seq=1) of whether or not this is true, but they tend to be written by non-native researchers, therefore muting a much needed context. Be mindful when researching this topic to note the origin of the researchers with respect to the colonies and colonizers they are writing about.

Reviewer: I'm wondering here if we want "the slave trade" or "slave trade(s)" as I think right now the grammar reading smoothly?

Author: good point. I was mostly thinking about the trans Atlantic slave trade, but I am not a historian - the slave trade(s) covers everything. I'll change that

> The practice of colonialism was most prominent with the British, Ottoman, and other European empires upon regions in the Americas, Asia, and Africa where slave trade was implemented as an economic vehicle. Contrary to popular belief, the colonialism also affected people of European descent, most notably, the Irish. Some studies have tried to [clarify the question](https://www.jstor.org/stable/20720270?seq=1) of whether or not this is true, but they tend to be written by non-native researchers, therefore muting a much needed context. Be mindful when researching this topic to note the origin of the researchers with respect to the colonies and colonizers they are writing about.

Reviewer: ```suggestion
The practice of colonialism was most prominent with the British, Ottoman, and other European empires upon regions in the Americas, Asia, and Africa where slave trade was implemented as an economic vehicle. Contrary to popular belief, the colonialism also affected people of European descent, most notably, the Irish. Some studies have tried to [clarify the question](https://www.jstor.org/stable/20720270?seq=1) of whether or not this is true, but they tend to be written by non-Native researchers, therefore muting a much needed context. Be mindful when researching this topic to note the origin of the researchers with respect to the colonies and colonisers that they are writing about.
```

Author: Will change this as "s" because majority. Glad you went over because "z" is American spelling and "s" is British spelling.

> An extension of colonialism that appears in the 21st century is neo-colonialism - a more pervasive form of imposing ideological and economic dominance on a country after it has been declared independent from its colonizer. The stealth nature of neo-colonialism can be traced to treaties, agreements, and ideologies that have been imparted from the oppresors on the oppressed because of previous dependencies.

Reviewer: I'm wondering if here we can break this out into neo-colonialism as its own term.

Author: Could be. I actually wanted to but didn't want to add a word that you hadn't already accounted for. Let me know if all it takes is a simple new doc and update and I'll break these out

---
title: colonialism
slug: colonialism
speech: noun
excerpt: the act of enforcing one's political and ideological dominance in a country or region that is not of one's origin

> sub_terms:
>     - text: modern
>     full_title: modern colonialism
>     - text: settler
>     full_title: settler colonialism
>     - text: Western
>     full_title: Western colonialism
> defined: true
> reading:
>     - text: A brief context of colonialism
>       href: https://plato.stanford.edu/entries/colonialism/
> ---

Reviewer: ```suggestion
sub_terms:
  - text: modern
    full_title: modern colonialism
  - text: settler
    full_title: settler colonialism
  - text: Western
    full_title: Western colonialism
defined: true
reading:
  - text: A brief context of colonialism
    href: https://plato.stanford.edu/entries/colonialism/
---
```

### Note

Though [often associated with imperialism](https://plato.stanford.edu/entries/colonialism/), the two words have different Latin etymologies that drive how the connotation of colonialism and the word colony have evolved. The most notable instances of colonialism to the general public are Western colonialism and recently, neo-colonialism.

The practice of colonialism was most prominent with the British, Ottoman, and other European empires upon regions in the Americas, Asia, and Africa where slave trade(s) was(were) implemented as an economic vehicle. Contrary to popular belief, the colonialism also affected people of European descent, most notably, the Irish. Some studies have tried to [clarify the question](https://www.jstor.org/stable/20720270?seq=1) of whether or not this is true, but they tend to be written by non-Native researchers, therefore muting a much needed context. Be mindful when researching this topic to note the origin of the researchers with respect to the colonies and colonisers that they are writing about.

> An extension of colonialism that appears in the 21st century is neo-colonialism - a more pervasive form of imposing ideological and economic dominance on a country after it has been declared independent from its colonizer. The stealth nature of neo-colonialism can be traced to treaties, agreements, and ideologies that have been imparted from the oppresors on the oppressed because of previous dependencies.

Reviewer: You are always welcome to include new words! Let's go ahead and break it out please, and we can work on that definition together. 👍🏽

Author: Okie! Onward. Thanks Tatiana
